packages/react-server/src/ReactFizzServer.js

```diff
@@ -2301,18 +2305,15 @@ function flushCompletedQueues(
       if (request.pendingRootTasks === 0) {
         if (enableFloat) {
           const preamble = request.preamble;
```

Comment:
It's probably time to rethink the concept of `request.preamble`. It was an elegant way to solve this problem before, but now it's way more complicated so it's not elegant and just unnecessary. The information is encoded on the responseState anyway.